src/cli.ts

```diff
@@ -169,6 +170,14 @@ async function run(args: CliArgs) {
   logger.log(`✨  Finished in ${formatDuration(duration)}`)
 
   await lintPackage(cwd)
+
+  if (result.length === 0) {
+    console.warn(
```

Comment:
it seems gonna log after "Finished" log, are we able to log this after entries are collected? we can count there and warning first

Reply:
### Current Behavior

```
devjiwonchoi@Jiwons-MacBook-Air equine % pnpm build

> equine@0.0.2 build /Users/devjiwonchoi/DevLife/OpenSource/equine
> bunchee

✨  Finished in 99ms
pkg.main is dist/index.js but the file does not exist.
pkg.types is dist/index.d.ts but the file does not exist.
```

Where do you want to place the log?

> are we able to log this after entries are collected?

Do you mean inside `bundle`?